internal/test/wmcb/wmcb_test.go

```diff
@@ -677,3 +677,48 @@ func (f *wmcbFramework) getLatestCniPluginsSHA() (string, error) {
 	sha512 := sha512AndFilename[0]
 	return sha512, nil
 }
+
+// initializeKubeNodePkgInfo populates the fileds of KubeNode pkg
+func (f *wmcbFramework) initializeKubeNodePkgInfo() error {
+	f.kubeNode = pkgInfo{}
+
+	if k8sVersion := framework.K8sVersion; k8sVersion == "" {
+		return fmt.Errorf("kubernetes version is empty")
+	}
+	f.kubeNode.url = kubeNodeBaseURL + framework.K8sVersion + "/kubernetes-node-windows-amd64.tar.gz"
+
+	kubeNodeSHA, err := f.kubeNode.getKubeNodeSHA()
+	if err != nil {
+		return fmt.Errorf("unable to get SHA for Kubernetes node binary for Windows: %v", err)
+	}
+	f.kubeNode.sha = kubeNodeSHA
+
+	f.kubeNode.shaType = "sha512"
+	return nil
+}
+
+// getKubeNodeSHA returns the SHA512 value of the kube node package.
```

Comment:
Please add a TODO and a story for this.

Reply:
Please explain what `this` is.

Reply:
I had an offline discussion with @VaishnaviHire. We decided to use `pkgInfo` struct for now for getting the SHA. We should ideally use `pkgInfo{}` as interface and have a slice of `pkgInfo{}` interfaces instead of have 2 different structs for the same `pkgInfo`

Reply:
@ravisantoshgudimetla this again is super low priority work that will never get picked up. We either do this now or not at all. There is a limit to which we can keep generating TODOs and future work.

Reply:
Ok, I am not sure if this blocks @VaishnaviHire for closing this ticket before tomorrow. If we don't mind, I think we should have this as part of the PR.

Reply:
I already added the TODOs. Should I include the refactoring as a part of this PR and implement the changes ?

Reply:
Please pair program.